apps/web/src/ee/billing/components/PlanHeader.tsx

```diff
@@ -136,7 +139,7 @@ export const PlanHeader = () => {
                   </Text>
                 </When>
               </Group>
-              <When truthy={!hasPaymentMethod}>
+              <When truthy={!isActive}>
```

Comment:
These checks need to be based on active subscription, otherwise customers which previously bought subscription won't be able to buy again because interval switcher and upgrade button were based on `hasPaymentMethod` which would be true for them.